crypto/ocsp/test/s2n/OCSP-TEST.md

```diff
@@ -0,0 +1,138 @@
+## Overview
```

Comment:
Nit: Why s2n test data files and README (e.g. `test/s2n/ocsp_cert.pem` and README) are copied here? I don't see these test data files (except the der array) are directly referenced. I saw the callout but think we may just need a comment with the s2n link.

Reply:
All the sha-256 OCSP response files were generated ourselves, so they don't exist in s2n. The README in s2n on how to generate the OCSP responses  also lacked some details on how to generate the `ocsp_response_wrong_signer.der` and how to generate `sha-256` responses. I think we should a runbook in aws-lc for future reference on how to regenerate new OCSP response tests.
Other than that, do you think I should directly read the files in this directory as part of the new code refactorization you mentioned above? That way we could remove the hard coded tests in `ocsp_test.cc` and having a separate folder for our test files would make sense.

Reply:
> All the sha-256 OCSP response files were generated ourselves, so they don't exist in s2n.

My miss. I thought they are the same. Thanks for explanation.

> directly read the files in this directory as part of the new code refactorization you mentioned above?

I don't have strong preference. If possible, I think just keeping one is good enough.